-- ALL CREDITS TO ARTICLE'S HUB, THANK HIM FOR THIS!!!
-- His github: https://github.com/Articles-Hub
local RequireService = cloneref or function(clone) return clone end
RS = RequireService(game:GetService("ReplicatedStorage"))
slaps = {
    ["Default"] = RS.b,
    ["Extended"] = RS.b,
    ------------------------------------------------------------------------
    ["ZZZZZZZ"] = RS.ZZZZZZZHit,
    ["Brick"] = RS.BrickHit,
    ["Snow"] = RS.SnowHit,
    ["Pull"] = RS.PullHit,
    ["Flash"] = RS.FlashHit,
    ["Spring"] = RS.springhit,
    ["Swapper"] = RS.HitSwapper,
    ["Bull"] = RS.BullHit,
    ["Dice"] = RS.DiceHit,
    ["Ghost"] = RS.GhostHit,
    ["Stun"] = RS.HtStun,
    ["Za Hando"] = RS.zhramt,
    ["Fort"] = RS.Fort,
    ["Magnet"] = RS.MagnetHIT,
    ["Pusher"] = RS.PusherHit,
    ["Anchor"] = RS.hitAnchor,
    ["Space"] = RS.HtSpace,
    ["Boomerang"] = RS.BoomerangH,
    ["Speedrun"] = RS.Speedrunhit,
    ["Mail"] = RS.MailHit,
    ["Golden"] = RS.GoldenHit,
    ["MR"] = RS.MisterHit,
    ["Reaper"] = RS.ReaperHit,
    ["Replica"] = RS.ReplicaHit,
    ["Defense"] = RS.DefenseHit,
    ["Killstreak"] = RS.KSHit,
    ["Reverse"] = RS.ReverseHit,
    ["Shukuchi"] = RS.ShukuchiHit,
    ["Duelist"] = RS.DuelistHit,
    ["woah"] = RS.woahHit,
    ["Ice"] = RS.IceHit,
    ["Adios"] = RS.hitAdios,
    ["Blocked"] = RS.BlockedHit,
    ["Engineer"] = RS.engiehit,
    ["Rocky"] = RS.RockyHit,
    ["Conveyor"] = RS.ConvHit,
    ["STOP"] = RS.STOP,
    ["Phantom"] = RS.PhantomHit,
    ["Wormhole"] = RS.WormHit,
    ["Acrobat"] = RS.AcHit,
    ["Plague"] = RS.PlagueHit,
    ["[REDACTED]"] = RS.ReHit,
    ["bus"] = RS.hitbus,
    ["Phase"] = RS.PhaseH,
    ["Warp"] = RS.WarpHt,
    ["Bomb"] = RS.BombHit,
    ["Bubble"] = RS.BubbleHit,
    ["Jet"] = RS.JetHit,
    ["Shard"] = RS.ShardHIT,
    ["potato"] = RS.potatohit,
    ["CULT"] = RS.CULTHit,
    ["bob"] = RS.bobhit,
    ["Buddies"] = RS.buddiesHIT,
    ["Spy"] = RS.SpyHit,
    ["Detonator"] = RS.DetonatorHit,
    ["Rage"] = RS.GRRRR,
    ["Trap"] = RS.traphi,
    ["Orbit"] = RS.Orbihit,
    ["Hybrid"] = RS.HybridCLAP,
    ["Slapple"] = RS.SlappleHit,
    ["Disarm"] = RS.DisarmH,
    ["Dominance"] = RS.DominanceHit,
    ["Link"] = RS.LinkHit,
    ["Rojo"] = RS.RojoHit,
    ["rob"] = RS.robhit,
    ["Rhythm"] = RS.rhythmhit,
    ["Nightmare"] = RS.nightmarehit,
    ["Hitman"] = RS.HitmanHit,
    ["Thor"] = RS.ThorHit,
    ["Retro"] = RS.RetroHit,
    ["Cloud"] = RS.CloudHit,
    ["Null"] = RS.NullHit,
    ["spin"] = RS.spinhit,
    ------------------------------------------------------------------------
    ["Poltergeist"] = RS.UTGHit,
    ["Clock"] = RS.UTGHit,
    ["Untitled Tag Glove"] = RS.UTGHit,
    ------------------------------------------------------------------------
    ["Kinetic"] = RS.HtStun,
    ["Recall"] = RS.HtStun,
    ["Balloony"] = RS.HtStun,
    ["Sparky"] = RS.HtStun,
    ["Boogie"] = RS.HtStun,
    ["Stun"] = RS.HtStun,
    ["Coil"] = RS.HtStun,
    ------------------------------------------------------------------------
    ["Diamond"] = RS.DiamondHit,
    ["Megarock"] = RS.DiamondHit,
    ------------------------------------------------------------------------
    ["Moon"] = RS.CelestialHit,
    ["Jupiter"] = RS.CelestialHit,
    ------------------------------------------------------------------------
    ["Mitten"] = RS.MittenHit,
    ["Hallow Jack"] = RS.HallowHIT,
    ------------------------------------------------------------------------
    ["OVERKILL"] = RS.Overkillhit,
    ["The Flex"] = RS.FlexHit,
    ["Custom"] = RS.CustomHit,
    ["God's Hand"] = RS.Godshand,
    ["Error"] = RS.Errorhit
}
for i,v in pairs(game.Players.LocalPlayer._unlockedGloves:GetChildren()) do
if v and not slaps[v] then
slaps[v] = RS.GeneralHit
end
end
return slaps
